rubygem-katello/rubygem-katello.spec

```diff
@@ -140,6 +140,10 @@ BuildRequires: %{?scl_prefix}rubygem(qpid_messaging) < 0.31.0
 BuildRequires: %{?scl_prefix_ruby}rubygems-devel
 BuildRequires: %{?scl_prefix_ruby}ruby(rubygems)
 
+#old dependencies that are no longer needed, remove in katello 2.6
```

Comment:
No, i was implying to leave this obsoletes in there for two releases and then remove them.  I could see an argument for just one release though.